blueprints/going-to-production-with-healthcare-apps.html.md

```diff
@@ -100,17 +100,17 @@ Your application will automatically be deployed to the regions specified and com
 
 ### Add API keys and credentials to the Secrets Manager
 
-Fly.io's [secret manager stores](https://fly.io/docs/reference/secrets/) your application's API credentials and secrets in a secure vault. The contents remain encrypted and are only available from running Machine instances as environment variables.
+Fly.io's [secrets manager stores](https://fly.io/docs/reference/secrets/) your application's API credentials and secrets in a secure vault. The contents remain encrypted and are only available from running Machine instances as environment variables.
 
 ### Talk to our solution architects if you have questions
 
 If at any time during your evaluation of Fly you have questions about the infrastructure, security, or service terms you can ask in the community forums or [email Fly.io with your questions](mailto:sales@fly.io).
 
 ## Deploy to production
 
-Once you've evaluated Fly.io and have a BAA, it's time to go to production. The remainder of this guide will walk you through how to setup an isolated production environment, control access, and deploy your application.
+Once you've evaluated Fly.io and have a BAA, it's time to go to production. The remainder of this guide will walk you through setting up an isolated production environment, controlling access, and deploying your application.
```

Comment:
curious why you suggest this change? I would leave as-is

Reply:
I have made the required change. Thank you for the review.